backend/app/Http/Controllers/AuthController.php

```diff
@@ -34,4 +34,25 @@ public function register(Request $request)
 
         return response($response, 201);
     }
+
+    public function login(Request $request)
+    {
+        $login_credentials = $request->validate([
+            'email' => ['required', 'string', 'email'],
+            'password' => ['required', 'string']
+        ]);
+
+        $user = User::where('email', $login_credentials['email'])->first();
+
+        if ($user != null) {
```

Comment:
```suggestion
        if ($user) {
```
since you are checking if there is a value you can use $user as parameter, this is a topic related to [truthy and falsy values](https://blog.devgenius.io/php-7-x-p5-truthy-bools-9e8917a26943).

Reply:
or you can create a guard clause
> guard clause simply a check that immediately exits the function, either with a return statement or an exception
```php
if (!$user) {
  return response()->json(['error' => "User doesn't exist"], 400);
}
```